docs/RELEASE.md

```diff
@@ -1,33 +1,57 @@
 # Release
 
-The outputs of a release are the `polkadot` and `polkadot-parachain` node binaries, the runtimes for Westend & Rococo
-and their system parachains, and new crate versions published to `crates.io`.
+The outputs of a stable release are:
+
+- The binaries:
+   - `polkadot`
+   - `polkadot-execute-worker`
+   - `polkadot-prepare-worker`
+   - `polkadot-parachain`
+   - `polkadot-omni-node`
+   - `chain-spec-builder`
+   - `frame-omni-bencher`
+
+   built for the `x86_64-unknown-linux-gnu` and `aarch64-apple-darwin` targets. The gpg signatures and sha256 checksums.
+
+- The runtimes for Westend and its system parachains
+- The new crate versions published to `crates.io`.
+
+
+# Timeline
+`Stable` releases are scheduled on a quarterly basis, usually by the end of the last month of each quarter. The exact schedule can be found on the [Release Registry](https://github.com/paritytech/release-registry/). It is possible to subscribe to a [calendar link](https://raw.githubusercontent.com/paritytech/release-registry/main/releases-v1.ics) to have it in your personal calendar.
+
+Each stable release is supported for a period of one year from its first release. For example, `Polkadot stable2412` was released on `2024-12-17` and its end of life is set to `2025-12-16`.
+
+During this period, each stable release is updated with patch releases, which are scheduled on a monthly basis and contain fixes for any bugs that may be found.
+
+ℹ️ Note: the binaries and runtimes (if needed) are only provided for the latest `stable' release, for the previous releases only the crates.io release takes place.
+
+This three month period between `stable` releases includes a 1.5 month QA period. This means that for each upcoming `stable` release, the branch from which that release will be made is created 1.5 months before the release date. This time is used to test the upcoming release candidate and find any issues that may arise with publishing crates to crates.io, binary artifacts and template synchronisation before the final release. The findings should be fixed and backported to the release branch. During this time, multiple release candidates may be built and rolled out.
 
 # Setup
 
-We have two branches: `master` and `stable`. `master` is the main development branch where normal Pull Requests are
-opened. Developers need to mostly only care about this branch.  
-The `stable` branch contains a version of the code that is ready to be released. Its contents are always audited.
-Merging to it is restricted to [Backports](#backports).
+We have two branches: `master` and `stableYYMM`. `master` is the main development branch where normal Pull Requests are
```

Comment:
Actually more than two now 😅

Reply:
Yep, this is more like a general concept, but I'll extend it😄